docs/simbad/query_tap.rst

```diff
@@ -0,0 +1,322 @@
+           h_link                                              hierarchy of membership measure
+      mesHerschel                                                   The Herschel observing Log
+           biblio                                                                 Bibliography
+         keywords                                                  List of keywords in a paper
+           mesXmm                                                           XMM observing log.
+    mesVelocities                                   Collection of HRV, Vlsr, cz and redshifts.
+           mesVar                         Collection of stellar variability types and periods.
+           mesRot                                               Stellar Rotational Velocities.
+            mesPM                                                Collection of proper motions.
+           mesPLX                                      Collection of trigonometric parallaxes.
+         otypedef                               all names and definitions for the object types
+           mesIUE                            International Ultraviolet Explorer observing log.
+           mesISO                              Infrared Space Observatory (ISO) observing log.
+          mesFe_h                  Collection of metallicity, as well as Teff, logg for stars.
+      mesDiameter                                             Collection of stellar diameters.
+      mesDistance                   Collection of distances (pc, kpc or Mpc) by several means.
+           otypes                           List of all object types associated with an object
+           mesSpT                                                Collection of spectral types.
+         journals                             Description of all used journals in the database
+
+To join tables, any columns sharing the same name are possible links between tables.
+To find the other possible joins, the `~astroquery.simbad.SimbadClass.list_linked_tables` method
+can be useful. Here we look for possible links with the ``mesDiameter`` table
+
+.. doctest-remote-data::
+
+    >>> from astroquery.simbad import Simbad
+    >>> Simbad.list_linked_tables("mesDiameter")
+    <Table length=1>
+     from_table from_column target_table target_column
+       object      object      object        object
+    ----------- ----------- ------------ -------------
+    mesDiameter      oidref        basic           oid
+
+The output indicates that the ``mesDiameter`` table can be linked to ``basic`` with the following
+join statement: ``[...] mesDiameter JOIN basic ON mesDiameter.oidref = basic.oid [...]``.
+
+
+.. graphviz:: simbad-er.gv
+    :layout: neato
+    :caption: A quick view of SIMBAD's tables. Hover the links to see the linked columns.
+    :alt: This interactive graph summarizes the information that can be obtained with `~astroquery.simbad.SimbadClass.list_tables` and `~astroquery.simbad.SimbadClass.list_linked_tables`.
```

Comment:
are the links working?  I don't see hovers.  But this renders nicely!
<img width="727" alt="image" src="https://github.com/astropy/astroquery/assets/143715/94ceea6e-01b0-4803-818b-51f5f52518da">

Reply:
is it possible to increase the font size though?  that might be a good idea, to match the overall fontsize of the document.  It's OK if that's not possible.

Reply:
Thanks for the review and all the English tips! The links work for me only when I use a local server to see the docs, there might be a bit of javascript to make it work in the sphinx-graphviz library, I'm not sure. I hope it will work too when it's live in astroquery's documentation. Otherwise it'll be a static graph and we'll have to change the legend. I'm looking for a way to change the font size :)

Reply:
A peek: 
![Peek 18-10-2023 15-45](https://github.com/astropy/astroquery/assets/16650466/1ec6ee03-2d07-4ef1-90b7-e0171839f7a7)

Reply:
The fontsize is changed... but in a bit of a ugly way. It has to be specified in each node in the DOT file and [it only works with pixels](https://graphviz.org/docs/attrs/fontsize/ "https://graphviz.org/docs/attrs/fontsize/"), not 'em'. I chose 16px as it seems to be the standard size of most browsers but it won't adapt to the readers' browsers parameters if they changed it.

For the hovering labels, I don't know. I tried a fresh python environment to generate the docs and it still worked. I noticed the tooltips only appear for me when I explicitly select the browser's window by clicking on it... Otherwise it might be an upstream bug with sphinx-graphviz?

Also, should I add in the comments of the `query_tap.rst` file that there is a script that can generate the file anew in case Simbad's schema changes? https://github.com/ManonMarchand/simbad-ER-diagram/blob/main/simbad-and-graphviz.py